conans/client/packager.py

```diff
@@ -24,14 +24,15 @@ def export_pkg(conanfile, package_id, src_package_folder, package_folder, hook_m
 
     copier = FileCopier(src_package_folder, package_folder)
     copier("*", symlinks=True)
-
-    copy_done = copier.report(output)
-    if not copy_done:
-        output.warn("No files copied from package folder!")
+    copier.report(output)
```

Comment:
I was suggesting further 😅 

- Remove the FileCopier.report functionality, completely
- Report copied files always from the manifest, summarizing it in a dict by extensions.

Reply:
Done!